orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/manifest/ManifestForceCacheRefreshTaskSpec.groovy

```diff
@@ -52,6 +52,6 @@ class ManifestForceCacheRefreshTaskSpec extends Specification {
     expect:
     taskResult.getStatus().isSuccessful()
     registry.counter("manifestForceCacheRefreshTask.refreshTimeout").count() == 1
-    registry.timer("manifestForceCacheRefreshTask.refreshComplete", "result", "success").count() == 1
+    registry.timer("manifestForceCacheRefreshTask.refreshComplete", "successful", "true").count() == 1
```

Comment:
The example you are baseing this on is an outlier within spinnaker. All the other microservices use "success".  Actually they use various things, but "success" is the most common and one I would like to standardize on (and am working on proposing).